### PR TITLE
[CARBONDATA-546] Extract data management command to carbon-spark-common module

### PR DESCRIPTION
**Pull request by jackylk**
Currently there are duplicated code for data management command in carbon-spark and carbon-spark2 module. In this PR, following commands are removed from `carbonTableSchema.scala` and extracted to carbon-spark-common:
- ShowLoads
- DeleteLoadById
- DeleteLoadByDate
- CleanFiles

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/265/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/266/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/278/

**Comment by chenliang613** [comment]
please add test cases to validate the below commands after extracting:

ShowLoads
DeleteLoadById
DeleteLoadByDate
CleanFiles

**Comment by jackylk** [comment]
@chenliang613 testcase added: CarbonCommandSuite

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/284/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/286/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/291/

**Comment by jackylk** [comment]
@chenliang613 please check

File: integration/spark2/src/main/scala/org/apache/spark/util/TableAPIUtil.scala [hunk 1]
```diff
@@ -51,4 +56,15 @@ object TableAPIUtil {
         .config(CarbonCommonConstants.STORE_LOCATION, storePath)
```
Comment (QiangCai) [inline integration/spark2/src/main/scala/org/apache/spark/util/TableAPIUtil.scala]: Can you remove this one?

Comment (QiangCai) [inline integration/spark2/src/main/scala/org/apache/spark/util/TableAPIUtil.scala]: BTW, now CarbonEnv will get  carbon.storelocation from CarobnProperties object.
So need to add storepath to configuration, but should add property to CarbonProperties Object. CarbonProperties.getInstance().addProperty(CarbonCommonConstants.STORE_LOCATION, storePath).

Comment (jackylk) [inline integration/spark2/src/main/scala/org/apache/spark/util/TableAPIUtil.scala]: master("local") is removed. 
Property is added

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/300/

**Comment by chenliang613** [comment]
Tested with DataManagementExample, ok.
LGTM

**Pull request closed without merging**